### PR TITLE
Contact information

### PR DESCRIPTION
**Pull request by Alzubair98**
requite to merge the final part of the portfolio project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -268,7 +323,45 @@ <h2 class="javaS">CSS</h2>
...
+        <form action="#" class="contact-information">
+          <input class="name-input" type="text" name="username" placeholder="alzubair" maxlength="30" />
+          <input
+            class="name-input"
+            type="email"
+            name="usermail"
+            placeholder="youremail@gmail.com"
+            required/>
+          
+          <textarea class="text-box" name="msg" id="message" cols="30" rows="10" maxlength="500">
```
Comment (m05e5) [inline index.html:359]: - [ ] Great job. Please as part of the requirement kindly add `required` to the text field and textarea also.
- [ ] Also the form should use the action and method attribute to integrate with [formspree service](https://formspree.io/html).

File: index.html [hunk 2]
```diff
@@ -268,7 +323,45 @@ <h2 class="javaS">CSS</h2>
...
+            Write your message here.</textarea>
+          
+        </form>
+        <button type="button" class="normal contact-btn">Get in touch</button>
```
Comment (m05e5) [inline index.html:363]: - [ ] The button should be part of your form with a `type='submit'` to submit the form

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @Alzubair98 , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [x] _**Please check the comments below**_

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -268,7 +323,45 @@ <h2 class="javaS">CSS</h2>
...
-</body>
\ No newline at end of file
+        <form action="https://formspree.io/f/contact-information" method="post" class="contact-information" id="contact-information" >
+          <input class="name-input" type="text" name="username" placeholder="alzubair" maxlength="30" required>
+          <input
+            class="name-input"
+            type="email"
+            name="usermail"
+            placeholder="youremail@gmail.com"
+            required>
```
Comment (zainsadaqat) [inline index.html:357]: [OPTIONAL]
This input field isn't properly closed. Kindly close the input field 👍🏻 
```
<input class="name-input"
            type="email"
            name="usermail"
            placeholder="youremail@gmail.com"
            required  />
```

File: index.html [hunk 4]
```diff
@@ -268,7 +323,45 @@ <h2 class="javaS">CSS</h2>
...
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
 
-</body>
\ No newline at end of file
+        <form action="https://formspree.io/f/contact-information" method="post" class="contact-information" id="contact-information" >
+          <input class="name-input" type="text" name="username" placeholder="alzubair" maxlength="30" required>
```
Comment (zainsadaqat) [inline index.html:351]: [OPTIONAL]
This input field isn't properly closed. Kindly close the input field 👍🏻

```
<input class="name-input" type="text" name="username" placeholder="alzubair" maxlength="30" required />
```

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @Alzubair98   😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

- [ ] [OPTIONAL] Please work on the responsiveness of your Portfolio 👍🏻 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**